### PR TITLE
Feat: improve cli results description panel

### PR DESCRIPTION
**Pull request by ArtsiomCh**
See commit messages for details.
Mostly refactoring CLI results UI building process and redesign UI by Andy's design and web results presentation as the baseline.

### CODE REVIEW

**Comment by ArtsiomCh** [comment]
See the `Build / Test` task failed:
```
> Task :compileKotlin FAILED
e: /home/runner/work/snyk-intellij-plugin/snyk-intellij-plugin/src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt: (219, 30): Unresolved reference: of
```
Looks like CI compiler can't see `java.nio.file.Path#of(java.lang.String, java.lang.String...)` for some reason. 
Locally everything compiles/runs well. 
@pavel-snyk please investigate

File: src/main/kotlin/io/snyk/plugin/ui/toolwindow/VulnerabilityDescriptionPanel.kt [hunk 1]
```diff
@@ -324,6 +325,13 @@ class VulnerabilityDescriptionPanel(private val vulnerability: Vulnerability) :
                 0,
                 false))
 
+        // don't remove that!
+        // Some magic (side-effect?) happens when JBHtmlEditorKit() initializing
+        // that make html tags like <em>, <p>, <ul> etc. be treated properly inside JEditorPane
+        // PS occasionally found that when played with com.intellij.util.ui.HtmlPanel
```
Comment (aldanchenko) [inline src/main/kotlin/io/snyk/plugin/ui/toolwindow/VulnerabilityDescriptionPanel.kt:350]: Don't sure about such comments in master 😉. But it conveys a thought well

File: src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt [hunk 2]
```diff
@@ -48,6 +49,8 @@ import javax.swing.tree.TreePath
 @Service
 class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
 
+    private val myAlarm = Alarm()
```
Comment (aldanchenko) [inline src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt:55]: Just as a suggestion: maybe rename it to something like scrollPaneAlarm to add more context to name.

File: src/main/kotlin/io/snyk/plugin/ui/toolwindow/VulnerabilityDescriptionPanel.kt [hunk 3]
```diff
@@ -47,33 +50,55 @@ class VulnerabilityDescriptionPanel(private val groupedVulns: Collection<Vulnera
         )
     }
 
+    private fun panelGridConstraints(row: Int) = baseGridConstraints(
```
Comment (aldanchenko) [inline src/main/kotlin/io/snyk/plugin/ui/toolwindow/VulnerabilityDescriptionPanel.kt:47]: Good idea 👍

File: src/main/kotlin/io/snyk/plugin/ui/toolwindow/VulnerabilityDescriptionPanel.kt [hunk 4]
```diff
@@ -278,50 +270,65 @@ class VulnerabilityDescriptionPanel(private val groupedVulns: Collection<Vulnera
...
+            component.foreground.blue,
+            alpha)
+    }
+
     private fun getOverviewPanel(): JPanel {
         val overviewPanel = JPanel()
-        overviewPanel.layout = GridLayoutManager(2, 1, Insets(0, 0, 0, 0), -1, 0)
+        overviewPanel.layout = GridLayoutManager(2, 1, Insets(0, 5, 0, 0), -1, 0)
 
         /*
```
Comment (aldanchenko) [inline src/main/kotlin/io/snyk/plugin/ui/toolwindow/VulnerabilityDescriptionPanel.kt:338]: Maybe you could delete this comment if we don't need it :)

**Review by aldanchenko — APPROVED** [review]
LGTM 👍 Thanks for your work! 🚀

File: src/main/kotlin/io/snyk/plugin/ui/toolwindow/VulnerabilityDescriptionPanel.kt (hunk 1 above)
Comment (ArtsiomCh) [reply src/main/kotlin/io/snyk/plugin/ui/toolwindow/VulnerabilityDescriptionPanel.kt:350]: At least it takes your attention and (and will for me as well few weeks/month later), that was the mean reason for such comment

File: src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt (hunk 2 above)
Comment (ArtsiomCh) [reply src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt:55]: done
PS In Idea source base it quite normal for inner objects to be named like my<Classname> and I found it helpful to get the idea about those objects scope while debugging.

**Pull request merged**